python/kserve/kserve/grpc/server.py

```diff
@@ -0,0 +1,40 @@
+from concurrent import futures
+import logging
+
+from kserve.grpc import grpc_predict_v2_pb2_grpc
+from kserve.grpc.servicer import InferenceServicer
+from kserve.handlers.dataplane import DataPlane
+
+from grpc import aio
+
+
+class GRPCServer:
+    def __init__(
+        self,
+        port: int,
+        data_plane: DataPlane
+    ):
+        self._port = port
+        self._data_plane = data_plane
+
+    def _create_server(self):
+        self._inference_servicer = InferenceServicer(self._data_plane)
+        self._server = aio.server(futures.ThreadPoolExecutor(max_workers=5))
```

Comment:
why setting to 5? can we make it configurable?

Reply:
I will update with param max_asyncio_workers.